kebechet/utils.py

```diff
@@ -91,9 +91,12 @@ def cloned_repo(manager: "ManagerBase", branch=None, **clone_kwargs):
 
     if _CLONE_DIRECTORY is not None:
         with cwd(_CLONE_DIRECTORY):
-            repo = _clone_repo_and_set_vals(
-                repo_url, _CLONE_DIRECTORY, branch, **clone_kwargs
-            )
+            try:
+                repo = _clone_repo_and_set_vals(
+                    repo_url, _CLONE_DIRECTORY, branch, **clone_kwargs
+                )
+            except git.GitCommandError:
+                repo = git.Repo(_CLONE_DIRECTORY)
```

Comment:
I'm wondering if this would work well for cases when git raises `GitCommandError` because of other issues. Would something like this cover cases when the repo is cloned?

```python
if os.path.isdir(os.path.join(_CLONE_DIRECOTRY, ".git")):
    repo = git.Repo(_CLONE_DIRECTORY)
else:
    repo = _clone_repo_and_set_vals(repo_url , _CLONE_DIRECTORY, branch, **clone_kwargs)
```

Reply:
Yeah, I'll change to using this method. Thanks for the suggestion Frido